application.md

```diff
@@ -0,0 +1,66 @@
+# GSoC Organization Application
+
+## Why does your org want to participate in GSoC?
+
+The Ruby org provides an umbrella org for both the Ruby language itself and for developer tools and other projects written in Ruby.
+We want to participate in order to grow the Ruby open source community, help students develop their programming and project management skills, and improve the experience of using Ruby for everyone.
+
+## How many potential mentors have agreed to mentor this year?
+
+So far, 10 developers have said they could potentially mentor this year.
+We hope to increase that number during the application period.
+
+## How will you keep mentors engaged with their students?
+
+Every mentor-student relationship is different, but our mentors stay in daily contact with their students via Slack.
+Students not only provide regular updates, but are encouraged to keep up an open line of communication with their mentors, asking questions when they are stuck and continuosly refining their project design (since no PRD survives contact with code).
+
+## How will you help your students stay on schedule to complete their projects?
+
+We expect students to be in daily communication with their mentors.
+
+## How will you get your students involved in your community during GSoC?
+
+Students join the project slack channel prior to submitting their applications.
+Mentors, other students, and other contributors are all available to the students from the community bonding period onwards, 
+
+## How will you keep your students involved in your community after GSoC?
+
+We do our best to get students work merged upstream before the end of the summer, giving them a sense of pride and accomplishment with their work on the project
+
+## Has your org been accepted as a mentor org in GSoC before?
+
+Yes.
+
+## Which years did you participate in GSoC?
+
+2014, 2015, 2016, 2018.
+
+## For each year your org has participated, provide the counts of successful and total students:
+
+2014:
+2015:
+2016:
+2018:
+
+## If your org has applied for GSoC before but not been accepted, select the years:
+
+None
```

Comment:
is this correct?

Reply:
yup, we have never been rejected. only Rails has been rejected some years.